src/celeritas/ext/GeantImporter.cc

```diff
@@ -129,6 +131,26 @@ struct ProcessFilter
     }
 };
 
+//---------------------------------------------------------------------------//
```

Comment:
Can we add a short documentation here? Also, I get the point of keeping it in `GeantImporter.cc`, but we have a combination of free functions and 2 classes in fairly large `.cc` file. So for readability we could break this into to 2 files (`GeantImporter` + `VolumeVisitor`). The `VolumeVisitor` makes sense after reading it, but since we're actually creating a map (instead of just visiting/reading the available volumes), so that we can provide a `vector<ImportVolume>` for `ImportData`, maybe the class name should be `ImportDataVolumesBuilder` or similar.

Reply:
Fair enough, I left the awkward file arrangement as-is to avoid all the code changes from moving it around, but since you've reviewed up till now that should be an easy-to-review change. Will do!